node-src/ui/tasks/verify.ts

```diff
@@ -32,15 +32,17 @@ export const runOnlyFiles = (ctx: Context) => ({
     ? `Snapshots will be limited to story files matching ${ctx.options.onlyStoryFiles
         .map((v) => `'${v}'`)
         .join(', ')}`
-    : `Snapshots will be limited to ${ctx.onlyStoryFiles.length} story files affected by recent changes`,
+    : `Snapshots will be limited to ${ctx.onlyStoryFiles?.length} story files affected by recent changes`,
 });
 
 export const runOnlyNames = (ctx: Context) => ({
   status: 'pending',
   title: 'Starting partial build',
-  output: `Snapshots will be limited to stories matching ${ctx.options.onlyStoryNames
-    .map((v) => `'${v}'`)
-    .join(', ')}`,
+  output: ctx.options.onlyStoryNames
+    ? `Snapshots will be limited to stories matching ${ctx.options.onlyStoryNames
+        .map((v) => `'${v}'`)
+        .join(', ')}`
+    : `Snapshots will be limited to ${ctx.onlyStoryFiles?.length} story files affected by recent changes`,
```

Comment:
The name of this function implies that `onlyStoryNames` is defined, so falling back to `onlyStoryFiles` is not only functionally wrong but also not necessary.